tsconfig.json

```diff
@@ -12,7 +12,6 @@
     "noImplicitAny": true,
     "importHelpers": true,
     "strictNullChecks": true,
-    "suppressImplicitAnyIndexErrors": true,
```

Comment:
This option got removed in Typescript 5.5. We also really should not have it enabled